controllers/edgedevicesignedrequest_controller.go

```diff
@@ -87,6 +87,9 @@ func (r *EdgeDeviceSignedRequestReconciler) Reconcile(ctx context.Context, req c
 			Labels: map[string]string{
 				v1alpha1.EdgeDeviceSignedRequestLabelName: v1alpha1.EdgeDeviceSignedRequestLabelValue,
 			},
+			Finalizers: []string{
+				DeviceFinalizer,
```

Comment:
Why is there `DeviceFinalizer` inside edsr controller?

Reply:
Because it creates the `EdgeDevice`.

Reply:
Yeah, make sense, sorry I should have open the whole file :)

Reply:
Yeah, make sense, sorry I should have open the whole file :)

Reply:
Yeah, make sense, sorry I should have open the whole file :)